main.tf

```diff
@@ -36,31 +39,41 @@ module "service_accounts" {
   depends_on  = [module.project_factory_project_services]
 }
 
-module "kms" {
-  # KMS is currently only used to encrypt pubsub queue. Disable it if we dont use it.
-  count               = var.use_internal_queue ? 0 : 1
+module "kms_generic_bucket" {
```

Comment:
suggestion (non-blocking): Let's** switch the name here to `kms_default_bucket`.